operator/controllers/seldondeployment_explainers.go

```diff
@@ -126,14 +126,19 @@ func (ei *ExplainerInitialiser) createExplainer(mlDep *machinelearningv1.SeldonD
 		httpPort = int(portNum)
 		customPort := getPort(portType, explainerContainer.Ports)
 
-		if p.Explainer.Endpoint != nil && p.Explainer.Endpoint.Type == machinelearningv1.GRPC {
-			explainerProtocol = "grpc"
+		if mlDep.Spec.Transport == machinelearningv1.TransportGrpc || (p.Explainer.Endpoint != nil && p.Explainer.Endpoint.Type == machinelearningv1.GRPC) {
+			explainerTransport = "grpc"
```

Comment:
Do we currently support GRPC explainers?

Reply:
Or was that added as an updated version of the conatiner? If so would that require a version bump on the deployment config?

Reply:
Yes we support grpc from explainer to model. The explainer call is http always.

Reply:
Oh right, from explainer to model, ok in that case it makes sense, from that context if seemed that it may be what the actual explainer was listening for. Makes sense.